plugins/guests/redhat/cap/change_host_name.rb

```diff
@@ -29,7 +29,13 @@ def self.change_host_name(machine, name)
               }
 
               # Restart network
-              service network restart
+              if test -f /etc/init.d/network; then
+                service network restart
+              elif systemctl -q is-enabled NetworkManager.service; then
```

Comment:
WDYT about `is-active` as opposed to `is-enabled` ?

Reply:
:+1: